looker/resource_looker_user_attribute_user.go

```diff
@@ -0,0 +1,154 @@
+package looker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	sdk "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+	"github.com/resolutionlife/terraform-provider-looker/internal/conv"
+)
+
+func resourceUserAttributeUser() *schema.Resource {
+	return &schema.Resource{
+		Description: "This resource sets a value onto a user for the given user attribute. If a default value is already set for the user attribute, this value will override the default value. Note that if the user attribute values are hidden (can be configured when provisioning a `looker_user_attribute`) then the provider does not have the permissions to read the hidden values, and cannot verify if the value has been manually changed in the Looker UI. The provider can however check if the value has been removed, and will prompt to recreate the resource.",
+
+		CreateContext: resourceUserAttributeUserCreate,
+		ReadContext:   resourceUserAttributeUserRead,
+		UpdateContext: resourceUserAttributeUserCreate,
```

Comment:
I like the clever re-use of create to also handle the value update here

Reply:
Thank you! ☺️